supervisord.conf

```diff
@@ -0,0 +1,19 @@
+; supervisor config file
```

Comment:
I'd recommend you copy the config in our Supervisor image here: https://github.com/praekeltfoundation/docker-supervisor

Also has some guides for setting up the individual programs.